### ISSUE TITLE
Issue with cactus-align-batch in pangenome workflow

### ISSUE THREAD

**Issue by RenzoTale88** [ComparativeGenomicsToolkit/cactus#615]
Hello,
I'm coming across an issue while running the cactus pangenome pipeline. I've followed the guidelines up the split of the alignments by chromosome successfully. However, when I try to run the `cactus-align-batch` stage on a single node in a GridEngine system with the following command line:
```
singularity exec --bind ${PWD}:/mnt docker://quay.io/comparative-genomics-toolkit/cactus:v2.0.4 cactus-align-batch \
    ./jobstore \
    ./PANGENOME/chroms/chromfile.txt \
    PANGENOME/align \
    --alignCores ${NSLOTS} \
    --alignOptions "--pafInput --pangenome --outVG --barMaskFilter 100000 --realTimeLogging --reference GENOME1 --workDir ${localdir}/tmp" \
    --maxCores ${NSLOTS} \
    --logFile align-all.log 
    --realTimeLogging \
    --workDir ./tmp 
```

I come across the following error:
```
[2021-12-06T05:53:09+0000] [MainThread] [I] [toil.leader] Issued job 'JobFunctionWrappingJob' kind-JobFunctionWrappingJob/instance-g60lovlz with job batch system ID: 33 and cores: 1, disk: 2.0 Gi, and memory: 2.0 Gi
[2021-12-06T05:53:12+0000] [MainThread] [I] [toil.worker] Redirecting logging to /exports/cmvm/eddie/eb/groups/prendergast_roslin/Andrea/GraphGenomes/Graph_122021/tmp/ccb234924f0759bb80d5ae7675e524cb/2ba7/worker_log.txt
[2021-12-06T05:53:14+0000] [MainThread] [I] [toil.leader] Finished toil run successfully.
[2021-12-06T05:53:14+0000] [MainThread] [I] [toil.realtimeLogger] Stopping real-time logging server.
[2021-12-06T05:53:14+0000] [MainThread] [I] [toil.realtimeLogger] Joining real-time logging server thread.
Traceback (most recent call last):
  File "/usr/local/bin/cactus-align-batch", line 8, in <module>
    sys.exit(main_batch())
  File "/usr/local/lib/python3.6/dist-packages/cactus/setup/cactus_align.py", line 802, in main_batch
    if options.batch:
AttributeError: 'Namespace' object has no attribute 'batch'
```
I've tried to add the `--batch` flag inside `--alignOptions`, but with no changes.

Any help would be much appreciated! Thanks in advance
Andrea

**Comment by glennhickey** [comment]
Ugh, looks like I only tested that on S3 output!  Sorry.  I will fix this but won't have time today to test it.  One thing to try is to just put an `options.batch = True` up on line 761.  Thanks for pointing this out!

**Comment by RenzoTale88** [comment]
Yes, that's what I've done this morning to see if it would work fine. I'm currently rerunning it, but good to know that I got the right solution :)

**Comment by RenzoTale88** [comment]
Hi, just found another issue:
```
[2021-12-09T23:31:44+0000] [MainThread] [I] [toil.leader] Finished toil run successfully.
[2021-12-09T23:31:44+0000] [MainThread] [I] [toil.realtimeLogger] Stopping real-time logging server.
[2021-12-09T23:31:45+0000] [MainThread] [I] [toil.realtimeLogger] Joining real-time logging server thread.
Traceback (most recent call last):
  File "/PATH/TO/Andrea/cactus-bin-v2.0.4/venv/bin/cactus-align-batch", line 8, in <module>
    sys.exit(main_batch())
  File "/PATH/TO/Andrea/cactus-bin-v2.0.4/venv/lib/python3.6/site-packages/cactus/setup/cactus_align.py", line 806, in main_batch
    if options.outVG:
AttributeError: 'Namespace' object has no attribute 'outVG'
```
I think it might simply be necessary to add `--outVG` and `--outGFA` to `main_batch()` at lines 750-751:
```
    parser.add_argument("--alignCoresOverrides", nargs="*", help = "Override align job cores for a chromosome. Space-separated list of chrom,cores pairse epxected")
    parser.add_argument("--outVG", action="store_true", help = "export pangenome graph in VG (.vg) in addition to HAL")
    parser.add_argument("--outGFA", action="store_true", help = "export pangenome grpah in GFA (.gfa.gz) in addition to HAL")
```
( I've set them to true in any case, just to be sure :) )

**Comment by glennhickey** [comment]
Yes, that will work.  Sorry again -- that's part of the same bug where the
codepath was only tested on s3 output which uses different logic to save
those files.

**Comment by glennhickey** [comment]
Should be resolved by #620
